Database-Files/CreateDatabase.sql

```diff
@@ -7,7 +7,7 @@ CREATE TABLE training (
     ID int PRIMARY KEY AUTO_INCREMENT,
     name VARCHAR(50) NOT NULL,
     description VARCHAR(200) NOT NULL,
-    trainingType VARCHAR(100)
+    trainingType VARCHAR(200)
 );
```

Comment:
Training should have a trainingLink field now

`trainingLink VARCHAR(200)`

Reply:
Will be added now with a link to general sharepoint